### ISSUE TITLE
Measuring and Harnessing Transference in Multi-Task Learning. (arXiv:2010.15413v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#131]
Measuring and Harnessing Transference in Multi-Task Learning. (arXiv:2010.15413v1 [cs.LG])<br>
https://ift.tt/3myLFZX<br>
<p>Multi-task learning can leverage information learned by one task to benefit the training of other tasks. Despite this capacity, na\"ive formulations often degrade performance and in particular, identifying the tasks that would benefit from co-training remains a challenging design question. In this paper, we analyze the dynamics of information transfer, or transference, across tasks throughout training. Specifically, we develop a similarity measure that can quantify transference among tasks and use this quantity to both better understand the optimization dynamics of multi-task learning as well as improve overall learning performance. In the latter case, we propose two methods to leverage our transference metric. The first operates at a macro-level by selecting which tasks should train together while the second functions at a micro-level by determining how to combine task gradients at each training step. We find these methods can lead to significant improvement over prior work on three supervised multi-task learning benchmarks and one multi-task reinforcement learning paradigm.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
